platform-sdk/platform-apps/tests/StatsSigningTestingTool/src/main/java/com/swirlds/demo/stats/signing/StatsSigningTestingToolState.java

```diff
@@ -115,15 +115,19 @@ public synchronized StatsSigningTestingToolState copy() {
     @Override
     public void preHandle(
             @NonNull final Event event,
-            @NonNull
-                    final Consumer<List<ScopedSystemTransaction<StateSignatureTransaction>>>
-                            stateSignatureTransactions) {
+            @NonNull final Consumer<ScopedSystemTransaction<StateSignatureTransaction>> stateSignatureTransactions) {
         final SttTransactionPool sttTransactionPool = transactionPoolSupplier.get();
         if (sttTransactionPool != null) {
             event.forEachTransaction(transaction -> {
                 if (transaction.isSystem()) {
                     return;
                 }
+
+                if (areTransactionBytesSystemOnes((ConsensusTransaction) transaction)) {
+                    stateSignatureTransactions.accept(
+                            new ScopedSystemTransaction(event.getCreatorId(), event.getSoftwareVersion(), transaction));
```

Comment:
You can return after this, since the application should not do anything with the system transactions other than invoke the callback.

Reply:
Good idea. Fixed